test/faker/index.js

```diff
@@ -257,15 +274,14 @@ class FakeServer extends EventEmitter {
                     options,
                 });
             });
-
             enableDestroy(this._server);
         });
     }
 
     close() {
         if (this._server) {
             return new Promise((resolve, reject) => {
-                this._server.destroy(err => {
+            this._server.destroy(err => {
```

Comment:
don't think this should have lost the indent

Reply:
it's because husky changed how it fires git hooks. Instead of using `package.scripts.precommit` you have to use something like `husky.hooks.pre-commit` 🤔

Reply:
Can we remove huskey?